observability/prometheus/prometheus.yml

```diff
@@ -17,6 +17,7 @@ scrape_configs:
   honor_timestamps: true
   scrape_interval: 1s
   scrape_timeout: 1s
+  # do we want one for /debug/prom/metrics?
```

Comment:
@stoksc

Reply:
yeah let's do it. people want this one and not the `det-state-metrics` pretty frequently.

maybe drop a comment above each block explaining the usage.